.github/workflows/python_package.yml

```diff
@@ -75,6 +77,7 @@ jobs:
     # yamllint disable-line rule:line-length
     name: ${{ matrix.task }} ${{ matrix.method }} (${{ matrix.os }}, ${{ matrix.compiler }}, Python ${{ matrix.python_version }})
```

Comment:
```suggestion
    name: ${{ matrix.task }} ${{ matrix.method }} (${{ matrix.os-display-name || matrix.os }}, ${{ matrix.compiler }}, Python ${{ matrix.python_version }})
```

Reply:
ah you are totally right! I keep forgetting to apply this pattern as I move jobs over, thanks for catching it.